src/frontend/src/lib/components/send/SendModal.svelte

```diff
@@ -60,6 +60,7 @@
 			}
 		}
 
+		// eslint-disable-next-line require-await
```

Comment:
why shall this be async?

Reply:
Because `loadTokenAndRun` requires the callback to be async. Few functions needs it to load data for example opening a modal that calls `autoLoadUserToken`.

Reply:
ah that's true! thank you!